platform/javascript/os_javascript.cpp

```diff
@@ -1005,6 +1126,7 @@ void OS_JavaScript::set_icon(const Ref<Image> &p_icon) {
 		link.href = url;
 	}, r.ptr(), len);
 	/* clang-format on */
+	r = PoolByteArray::Read();
```

Comment:
This isn't necessary here since the read is deleted per RAII at the end of the function.
In `set_custom_mouse_cursor`, the read isn't used at the end of the function, so it blocks the pool unnecessarily.

Reply:
Ok, thanks for explanation :+1:.